lib/network/rest/RESTServer.js

```diff
@@ -51,7 +51,7 @@ function sendError(res, log, error, optMessage) {
  */
 function parseURL(urlStr, expectKey) {
```

Comment:
Would be good to have a couple unit tests for `parseURL` with special characters

Reply:
@jonathan-gramain I moved `parseURL` from `RESTServer.js` to `utils.js` and added a new tests file `utils.js` with tests.